fs2-google-pubsub-http/src/main/scala/com/permutive/pubsub/consumer/http/internal/Model.scala

```diff
@@ -32,21 +37,44 @@ private[http] object Model {
     maxMessages: Int
   )
 
+  object PullRequest {
+    implicit final val PullRequestCodec: JsonValueCodec[PullRequest] =
+      JsonCodecMaker.make[PullRequest](CodecMakerConfig)
+  }
+
   case class PullResponse(
     receivedMessages: List[ReceivedMessage]
   )
 
+  object PullResponse {
+    implicit final val PullResponseCodec: JsonValueCodec[PullResponse] =
+      JsonCodecMaker.make[PullResponse](CodecMakerConfig)
+  }
+
   case class ReceivedMessage(
     ackId: AckId,
     message: PubsubMessage
   )
 
+  object ReceivedMessage {}
```

Comment:
Empty object here for some reason?

Reply:
Just thought I'll need to make an encoder for it, but I didn't! I'll remove it.